public/js/src/client.js

```diff
@@ -566,4 +591,14 @@ $(document).ready(function(){
 
 	// Begins the show!
 	setup();
-});
\ No newline at end of file
+});
+
+/* 
+    I would recommend breaking this into discreet commonJs modules, 
+    and using Browserify to compile. This would mean attaching each 
+    externally accessed method to an object, and concluding your file
+    with a `module.exports = yourObjectName`.  
+
+    This makes unit tests and documentation easier, and is a relief 
+    for future developers.  It also lets me organize my components.
+ */
```

Comment:
Can we talk about this at our meeting?  Even after looking over commonJs/requireJs/Broswerify, I have a couple of questions, about how it would work. Do you have an example of a site or project that you have done that has used any of these technologies? I would love to take a look.  I am actually starting a new project RN that will hopefully be really javascript intensive (I think you are going to really like it, btw), and it would be nice to have a good semantic separation of the 200+ functions I am going to need. I don't have a good sense for how to break down JS files in ways that are encouraged. YET!